bse/bsesong.cc

```diff
@@ -497,8 +497,10 @@ bse_song_ensure_master (BseSong *self)
     {
       BseUndoStack *ustack = bse_item_undo_open (self, "Create Master");
       child = (BseSource*) bse_container_new_child_bname (BSE_CONTAINER (self), BSE_TYPE_BUS, master_bus_name(), NULL);
-      g_object_set (child, "master-output", TRUE, NULL); // not-undoable
       Bse::BusImpl *bus = child->as<Bse::BusImpl*>();
+      bus->block_property_undo();
```

Comment:
Get rid of the block/unblock here, I'd like this to be handled elsewhere.

Reply:
Ok, I removed the block/unblock stuff now.